### PR TITLE
add article about localization basics

### PR DESCRIPTION
**Pull request by marketachalupnikova**

### CODE REVIEW

File: blog/2022-04-05-localization basics S01E01.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+description: "Localization basics S01E01: what are t9n, l10n, i18n, g11n? Learn about translation, software localization, internationalization and globalization"  
+
+
+tags: [i18n, l10n, t9n, g11n, translation, localization, internationalization, globalization]
+---
+
+![Localization basics S01E01: What are t9n, l10n, i18n, g11n? New secret language?](/img/blog/t9n_l10n_i18n_localization_internationalization.png)
+
+In the computing world, you may come across a few confusing acronyms. Those number-based words are called **numeronyms**. They are often used to abbreviate long words. Just take the first and last letter and count the letters in between.
```
Comment (JanCizmar) [inline blog/2022-04-05-localization basics S01E01.md:19]: computer world?

File: blog/2022-04-05-localization basics S01E01.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+## What is l10n?
+Localization refers to the adaptation of a product, app, or content to meet the language, cultural and other requirements of a specific target market (a "locale") to be available to your target audience. 
+
+The process of software localization can take **three different forms**:
+
+* **Waterfall Localization** - Quick time-to-market in your source language because localization begins after all the features had been implemented and the product is released.
+
+* **Agile Localization** - Increased productivity and flexibility in your localization workflow. Localization is incorporated into an agile product development cycle. It is accomplished in short work cycles (sprints).
+
+* **Continuous Localization** - Improved version of Agile localization: continuous localization is an ongoing cycle. Get feedback from all stakeholders faster and 	reduce the risk of localization errors by launching in many markets at the same time.
```
Comment (JanCizmar) [inline blog/2022-04-05-localization basics S01E01.md:41]: and ?? reduce

**Review by JanCizmar — APPROVED** [review]

File: blog/2022-04-05-localization basics S01E01.md (hunk 1 above)
Comment (stepan662) [reply blog/2022-04-05-localization basics S01E01.md:19]: tech world?

File: blog/2022-04-05-localization basics S01E01.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+## What is i18n? 
+Internationalization is not the same as software localization.
+
+*“Internationalization is the process of designing and developing a product or application so that it may be localized for different cultures, regions, and languages. To put it another way, internationalization is what makes localization possible.”*
+
+I18n is the first step in bringing software global. Software developers are usually in charge of internationalization, whereas translators are in charge of localization. Before you start working on any localization tasks, make sure you finish any internationalization work first.
```
Comment (stepan662) [inline blog/2022-04-05-localization basics S01E01.md:48]: bringing -> making?

File: blog/2022-04-05-localization basics S01E01.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+Internationalization is not the same as software localization.
+
+*“Internationalization is the process of designing and developing a product or application so that it may be localized for different cultures, regions, and languages. To put it another way, internationalization is what makes localization possible.”*
+
+I18n is the first step in bringing software global. Software developers are usually in charge of internationalization, whereas translators are in charge of localization. Before you start working on any localization tasks, make sure you finish any internationalization work first. 
+
+
+Internationalization entails a **variety of tasks**, including:
+
+* **Text length** - When designing software code, developers can account for the differences in length between languages to guarantee that translated text is visible in software interfaces, even if it is longer than the original.
```
Comment (stepan662) [inline blog/2022-04-05-localization basics S01E01.md:53]: designing software code -> designing software

File: blog/2022-04-05-localization basics S01E01.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+* **Currencies symbols and numbers** - Each language has its own system for writing numbers and currency, and software should be able to handle all of them.
+
+* **Date and time formats** - Different countries use different date and time formats, therefore software should account for all of them.
+
+* **Character encoding** - The internationalization should be able to handle a wide range of scripts, even some that aren't Latin.
```
Comment (stepan662) [inline blog/2022-04-05-localization basics S01E01.md:59]: scripts? -> probably "characters"

**Review by stepan662 — APPROVED** [review]

**Pull request merged**